subsys/bluetooth/host/l2cap_br_interface.h

```diff
@@ -37,3 +40,17 @@ void l2cap_br_encrypt_change(struct bt_conn *conn, uint8_t hci_status);
 
 /* Handle received data */
 void bt_l2cap_br_recv(struct bt_conn *conn, struct net_buf *buf);
+
```

Comment:
could you move this whole chunk back to `l2cap_br.h`?
I'd like to keep `l2cap_br_interface.h` only for functions that are called from `l2cap.c`.

Reply:
I did not find file `l2cap_br.h`. Do you mean it is file `l2cap_br_internal.h`? If yes, I think we can move line 44 ~ line 56 to file `l2cap_br_internal.h`.